terraform/production/main.tf

```diff
@@ -127,7 +127,7 @@ module "redis" {
 
   prefix                  = local.prefix
   ecs_security_group_id   = module.application.ecs_security_group_id
-  node_type               = "cache.t2.micro"
+  node_type               = "cache.t4g.micro"

```

Comment:
Just updating prod and dev now as well given they're currently set to the same `node_type` as staging. This can be reviewed later of course.